src/components/alert/alert.stories.svelte

```diff
@@ -78,7 +78,7 @@
 </Story>
 
 <Meta
-  title="Alert"
+  title="Components/Alert"
```

Comment:
The sorting sucks :laughing: Maybe they've fixed it in one of the new versions :laughing: